make.paws/inst/extdata/packages.yml

```diff
@@ -412,16 +451,19 @@
   services:
     - connect
     - connectcampaigns
+    - connectcases
     - connect-contact-lens
     - connectparticipant
+    - email
+    - entityresolution
```

Comment:
This one is tough. Maybe in analytics since it is for determining which database records are really referring to the same thing. I don’t think it belongs in customer engagement though.

Reply:
I found this one hard as well. I initially thought customer due to the description.

> With Entity Resolution, you can match source records containing consumer identifiers, such as name, email address, and phone number. This is true even when these records have incomplete or conflicting identifiers. 

But more that happy to move it over to analytics